jobs/signing/sign-artifacts/Jenkinsfile

```diff
@@ -206,12 +205,16 @@ node {
                 //
                 // where <RELEASE-NAME> is something like '4.1.0-rc.5'
 
-                // transform the file into a directory containing the file
+                // transform the file into a directory containing the
+                // file, mirror to google
                 commonlib.shell('''
                     for file in sha256=*; do
-                        mv $file signature-1
+                        mv $file ${params.SIGNATURE_NAME}
                         mkdir $file
-                        mv signature-1 $file
+                        mv ${params.SIGNATURE_NAME} $file
+                        until gsutil cp -n -v -L cp.log -r $file gs://openshift-release/official/signatures/openshift/release; do
```

Comment:
should probably terminate at some point if it's not working?

Reply:
where da creds at?

Reply:
re: creds: It's one of those `gsutil login` type things.

re: terminate: yeah.. I suppose so? I could add an increment and then `break` after it exceeds like.... 10 tries or something.